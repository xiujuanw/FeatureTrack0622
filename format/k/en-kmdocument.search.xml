<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<recordset table="format">
  <record id="syslanguage=&quot;en&quot; and name=&quot;kmdocument.search&quot;" recordid="en - kmdocument.search">
    <name mandatory="true" type="string">kmdocument.search</name>
    <field sctype="array">
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">1</line>
        <column type="decimal">1</column>
        <length type="decimal">88</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">Search kmdocument records</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=Search kmdocument records	Name=Label0	Opaque=1	BackColor=6	Bold=1	FontIncrease=2	ForeColor=10	Height=2	Name=Label0	Elastic=1	Width=176	X=0	Y=0	Caption=Search kmdocument records		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">3</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">id:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=id:	Name=Label30	ForeColor=0	Height=2	Name=Label30	Width=34	X=2	Y=4		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">3</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">id</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text1	Height=2	Name=Text1	Width=46	X=36	Y=4		Input=id	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">4</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">title:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=title:	Name=Label29	ForeColor=0	Height=2	Name=Label29	Width=34	X=2	Y=6		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">4</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">title</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text2	Height=2	Name=Text2	Width=46	X=36	Y=6		Input=title	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">5</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">summary:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=summary:	Name=Label28	ForeColor=0	Height=2	Name=Label28	Width=34	X=2	Y=8		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">5</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">summary</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text3	Height=2	Name=Text3	Width=46	X=36	Y=8		Input=summary	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">6</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">doctype:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=doctype:	Name=Label27	ForeColor=0	Height=2	Name=Label27	Width=34	X=2	Y=10		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">6</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">doctype</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text4	Height=2	Name=Text4	Width=46	X=36	Y=10		Input=doctype	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">7</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">categories:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=categories:	Name=Label6	ForeColor=0	Height=2	Name=Label6	Width=34	X=2	Y=12		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">512</flags>
        <line type="decimal">7</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">2</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input type="string">categories</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">MultiText;Name=MultiText5	Height=4	Name=MultiText5	Width=46	X=36	Y=12		Input=categories	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">9</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">problem:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=problem:	Name=Label26	ForeColor=0	Height=2	Name=Label26	Width=34	X=2	Y=16		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">9</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">problem</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text7	Height=2	Name=Text7	Width=46	X=36	Y=16		Input=problem	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">10</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">solution:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=solution:	Name=Label25	ForeColor=0	Height=2	Name=Label25	Width=34	X=2	Y=18		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">10</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">solution</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text8	Height=2	Name=Text8	Width=46	X=36	Y=18		Input=solution	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">11</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">error:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=error:	Name=Label24	ForeColor=0	Height=2	Name=Label24	Width=34	X=2	Y=20		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">11</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">error</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text9	Height=2	Name=Text9	Width=46	X=36	Y=20		Input=error	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">12</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">fix:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=fix:	Name=Label23	ForeColor=0	Height=2	Name=Label23	Width=34	X=2	Y=22		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">12</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">fix</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text10	Height=2	Name=Text10	Width=46	X=36	Y=22		Input=fix	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">13</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">cause:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=cause:	Name=Label22	ForeColor=0	Height=2	Name=Label22	Width=34	X=2	Y=24		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">13</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">cause</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text11	Height=2	Name=Text11	Width=46	X=36	Y=24		Input=cause	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">14</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">reference:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=reference:	Name=Label21	ForeColor=0	Height=2	Name=Label21	Width=34	X=2	Y=26		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">14</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">reference</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text12	Height=2	Name=Text12	Width=46	X=36	Y=26		Input=reference	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">15</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">question:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=question:	Name=Label20	ForeColor=0	Height=2	Name=Label20	Width=34	X=2	Y=28		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">15</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">question</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text13	Height=2	Name=Text13	Width=46	X=36	Y=28		Input=question	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">16</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">answer:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=answer:	Name=Label19	ForeColor=0	Height=2	Name=Label19	Width=34	X=2	Y=30		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">16</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">answer</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text14	Height=2	Name=Text14	Width=46	X=36	Y=30		Input=answer	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">17</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">status:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=status:	Name=Label18	ForeColor=0	Height=2	Name=Label18	Width=34	X=2	Y=32		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">17</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">status</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text15	Height=2	Name=Text15	Width=46	X=36	Y=32		Input=status	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">18</line>
        <column type="decimal">2</column>
        <length type="decimal">17</length>
        <window type="decimal">0</window>
        <attribute type="decimal">2</attribute>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">usage.count:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=usage.count:	Name=Label17	ForeColor=0	Height=2	Name=Label17	Width=34	X=2	Y=34		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">18</line>
        <column type="decimal">19</column>
        <length type="decimal">23</length>
        <window type="decimal">0</window>
        <attribute type="decimal">1</attribute>
        <graph sctype="array"/>
        <input type="string">usage.count</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=Text16	Height=2	Name=Text16	Width=46	X=36	Y=34		Input=usage.count	</property>
      </field>
    </field>
    <file.name type="string">kmdocument</file.name>
    <editor type="string">jantonecchia</editor>
    <last.update type="dateTime">09/28/05 03:19:05</last.update>
    <help.topic NullValue="1" type="string"/>
    <syslanguage type="string">en</syslanguage>
    <sysmodcount type="decimal">0</sysmodcount>
    <sysmoduser type="string">falcon</sysmoduser>
    <sysmodtime type="dateTime">10/25/05 03:23:26</sysmodtime>
    <description NullValue="1" type="string"/>
  </record>
</recordset>
