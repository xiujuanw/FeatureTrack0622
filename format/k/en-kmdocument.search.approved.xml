<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<recordset table="format">
  <record id="syslanguage=&quot;en&quot; and name=&quot;kmdocument.search.approved&quot;" recordid="en - kmdocument.search.approved">
    <name mandatory="true" type="string">kmdocument.search.approved</name>
    <field sctype="array">
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">1</line>
        <column type="decimal">3</column>
        <length type="decimal">50</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">Search Approved Documents</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=Search Approved Documents	Name=label1138233346263	Bold=1	?Caption=	?FontIncrease=	FontIncrease=5	Height=2	Name=label1138233346263	Width=100	X=4	Y=0		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">2</line>
        <column type="decimal">3</column>
        <length type="decimal">8</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">Title:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=Title:	Name=label1126749508563	Height=2	Name=label1126749508563	Width=15	X=4	Y=3		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">2</line>
        <column type="decimal">14</column>
        <length type="decimal">33</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input type="string">title</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=input1126749490859	?ReadOnly=[status]="retired"	Height=2	Name=input1126749490859	Mandatory=0	Width=65	X=26	Y=3		Input=title	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">2</line>
        <column type="decimal">53</column>
        <length type="decimal">11</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">Document ID:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=Document ID:	Name=label1132674234921	Height=2	Name=label1132674234921	Width=21	X=104	Y=3		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">2</line>
        <column type="decimal">64</column>
        <length type="decimal">15</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input type="string">id</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=input1132674266481	Height=2	Name=input1132674266481	ReadOnly=0	Width=30	X=126	Y=3		Input=id	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">2</line>
        <column type="decimal">82</column>
        <length type="decimal">5</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">VO</output>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Height=2	Name=	Visible=0	Width=9	X=162	Y=3		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">4</line>
        <column type="decimal">3</column>
        <length type="decimal">8</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">Summary:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=Summary:	Name=label1126749559861	Height=2	Name=label1126749559861	Width=16	X=4	Y=6		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">4</line>
        <column type="decimal">14</column>
        <length type="decimal">33</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input type="string">summary</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">Text;Name=input1195000731172	Height=2	Name=input1195000731172	Width=65	X=26	Y=6		Input=summary	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">4</line>
        <column type="decimal">53</column>
        <length type="decimal">11</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">Document Type:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=Document Type:	Name=label1126749934554	?Caption=	Height=2	Name=label1126749934554	Width=21	X=104	Y=6		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">4096</flags>
        <line type="decimal">4</line>
        <column type="decimal">64</column>
        <length type="decimal">15</length>
        <window type="decimal">1</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input type="string">doctype</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">ComboBox;Height=2	Name=	SelectOnly=1	Width=30	X=126	Y=6		Input=doctype	DisplayList=$G.kmdoctype.display	ValueList=$G.kmdoctype.name	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">4096</flags>
        <line type="decimal">5</line>
        <column type="decimal">1</column>
        <length type="decimal">48</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">VO</output>
        <format NullValue="1" type="string"/>
        <property type="string">Frame;Name=fieldset1398055807382	ArrayLength=0	Height=17	Name=fieldset1398055807382	Visible=0	Width=96	X=0	Y=9		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">4096</flags>
        <line type="decimal">6</line>
        <column type="decimal">3</column>
        <length type="decimal">44</length>
        <window type="decimal">5</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">VO</output>
        <format NullValue="1" type="string"/>
        <property type="string">Table;ColumnHeadings=1	Columns=column1138726658377	DefaultButtonID=0	Height=9	Name=	TabStop=19	Width=87			X=4	Y=10		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">4100</flags>
        <line type="decimal">1</line>
        <column type="decimal">1</column>
        <length type="decimal">4</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input type="string">generatedcats</input>
        <output type="string">Categories</output>
        <format NullValue="1" type="string"/>
        <property type="string">Column;		Caption=Categories	=		Column Width%=100.0					ValueList=categories	DisplayList=generatedcats	Input=generatedcats	Y=0	X=0	Width=8	SelectOnly=1	ReadOnly=1	Name=	ShowTitle=0	ClickableTitle=0	scType=ComboBox		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">4096</flags>
        <line type="decimal">1</line>
        <column type="decimal">1</column>
        <length type="decimal">1</length>
        <window type="decimal">1</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">VO</output>
        <format NullValue="1" type="string"/>
        <property type="string">end;X=1	Y=1	Height=2	Width=2	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">4096</flags>
        <line type="decimal">1</line>
        <column type="decimal">1</column>
        <length type="decimal">1</length>
        <window type="decimal">1</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">VO</output>
        <format NullValue="1" type="string"/>
        <property type="string">end;X=1	Y=1	Height=2	Width=2	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">4096</flags>
        <line type="decimal">11</line>
        <column type="decimal">10</column>
        <length type="decimal">12</length>
        <window type="decimal">1</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">Add</output>
        <format NullValue="1" type="string"/>
        <property type="string">Button;Caption=Add	ButtonID=318	?Bitmap=	Height=2	Name=	TabStop=20	Width=24			X=18	Y=21		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">4096</flags>
        <line type="decimal">11</line>
        <column type="decimal">26</column>
        <length type="decimal">12</length>
        <window type="decimal">1</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">Remove</output>
        <format NullValue="1" type="string"/>
        <property type="string">Button;Caption=Remove	ButtonID=319	Height=2	Name=	TabStop=21	Width=24			X=50	Y=21		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">5</line>
        <column type="decimal">53</column>
        <length type="decimal">11</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">Status:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=Status:	Height=2	Name=	Width=21	X=104	Y=9		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">4096</flags>
        <line type="decimal">5</line>
        <column type="decimal">64</column>
        <length type="decimal">15</length>
        <window type="decimal">1</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input type="string">status</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">ComboBox;Height=2	Name=	SelectOnly=0	Width=30	X=126	Y=9		Input=status	DisplayList=Internally Approved;Externally Approved	ValueList=internal;external	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">7</line>
        <column type="decimal">53</column>
        <length type="decimal">11</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">Locale:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=Locale:	?Caption=	Height=2	Name=	Width=22	X=104	Y=12		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">4096</flags>
        <line type="decimal">7</line>
        <column type="decimal">64</column>
        <length type="decimal">15</length>
        <window type="decimal">1</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input type="string">locale</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">ComboBox;Height=2	Name=	SelectOnly=1	Width=30	X=126	Y=12		Input=locale	DisplayList=$G.language.km.names	ValueList=$G.language.km.ids	</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">0</flags>
        <line type="decimal">8</line>
        <column type="decimal">53</column>
        <length type="decimal">11</length>
        <window type="decimal">0</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input NullValue="1" type="string"/>
        <output type="string">Author:</output>
        <format NullValue="1" type="string"/>
        <property type="string">Label;Caption=Author:	?Caption=	Height=2	Name=	Width=22	X=104	Y=15		</property>
      </field>
      <field sctype="structure">
        <flags type="decimal">4096</flags>
        <line type="decimal">8</line>
        <column type="decimal">64</column>
        <length type="decimal">15</length>
        <window type="decimal">1</window>
        <attribute NullValue="1" type="decimal"/>
        <graph sctype="array"/>
        <input type="string">author</input>
        <output NullValue="1" type="string"/>
        <format NullValue="1" type="string"/>
        <property type="string">ComFill;Name=comfill1195000067973	dataType=string	Height=2	Name=comfill1195000067973	FillButtonID=9	ThirdButtonID=8	ComboButtonVisible=0	ThirdButtonVisible=1	Width=30	X=126	Y=15		Input=author	</property>
      </field>
    </field>
    <file.name NullValue="1" type="string"/>
    <editor type="string">yuliang.gao</editor>
    <last.update type="dateTime">07/14/14 21:55:50</last.update>
    <help.topic NullValue="1" type="string"/>
    <syslanguage type="string">en</syslanguage>
    <sysmodcount type="decimal">22</sysmodcount>
    <sysmoduser type="string">yuliang.gao</sysmoduser>
    <sysmodtime type="dateTime">07/14/14 21:55:50</sysmodtime>
    <description NullValue="1" type="string"/>
  </record>
</recordset>
